monitor/utils/monasca/connector.py

```diff
@@ -14,11 +14,6 @@
 # limitations under the License.
 
 from monascaclient import exc
-import ConfigParser
```

Comment:
This is a unused import? If yes, please comment this modification in the description of this PR. This issue repeats itself through all file.